package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "hafas-client": "^5.8.0",
     "moment": "^2.27.0",
-    "vbb-line-colors": "^1.0.2",
+    "vbb-line-colors": "^1.0.3",
```

Comment:
Could you please change vbb-line-colors to `1.0.4`?

After that, I will pull your changes :)

Reply:
Done.